src/Common/src/Interop/Windows/NtDll/Interop.RtlGetVersion.cs

```diff
@@ -11,12 +11,17 @@ internal partial class NtDll
         [DllImport(Libraries.NtDll, ExactSpelling=true)]
         private static extern int RtlGetVersion(ref RTL_OSVERSIONINFOEX lpVersionInformation);
 
-        internal static unsafe string RtlGetVersion()
+        internal static unsafe int RtlGetVersionEx(out RTL_OSVERSIONINFOEX osvi)
```

Comment:
I realize it was already like this, but does this actually need to be `unsafe`?

Reply:
To use `sizeof`, yes. And it is used to count the struct with fixed size buffer.